### PR TITLE
ST-1175: Add confluent server docker image

### PR DESCRIPTION
**Pull request by omkreddy**

### CODE REVIEW

**Comment by omkreddy** [comment]
I will add the tests in subsequent PRs.

File: debian/enterprise-kafka/Dockerfile [hunk 1]
```diff
@@ -46,3 +45,5 @@ VOLUME ["/var/lib/${COMPONENT}/data", "/etc/${COMPONENT}/secrets"]
 COPY include/etc/confluent/docker /etc/confluent/docker
 
 CMD ["/etc/confluent/docker/run"]
+
+ONBUILD RUN echo 'WARNING: enterprise-kafka image is now deprecated!. Please use cp-server image.'
```
Comment (ijuma) [inline debian/enterprise-kafka/Dockerfile]: Not sure we want these warnings, let me check with @addisonhuddy.

Comment (ijuma) [reply debian/enterprise-kafka/Dockerfile]: @omkreddy Let's remove the warnings.

Comment (omkreddy) [reply debian/enterprise-kafka/Dockerfile]: Ok. Deprecation was was mentioned in one-pager.  will remove.

Comment (addisonhuddy) [reply debian/enterprise-kafka/Dockerfile]: @ijuma and I chatted and we agree that the warning is a bit aggressive in this first release.  We will add a deprecation warning in the README for this first release, and add the ONBUILD warning later.

**Pull request merged**